nifi-nar-bundles/nifi-azure-bundle/nifi-azure-processors/src/main/java/org/apache/nifi/processors/azure/storage/utils/AzureStorageUtils.java

```diff
@@ -70,7 +74,7 @@
                     "In addition, the provenance repositories may be put on encrypted disk partitions.")
             .addValidator(StandardValidators.NON_EMPTY_VALIDATOR)
             .expressionLanguageSupported(ExpressionLanguageScope.FLOWFILE_ATTRIBUTES)
-            .required(true)
+            .required(false)
```

Comment:
I'd update the description to add suggestion for using the Lookup service to make credential dynamic.